### PR TITLE
Bugfix/upgrade linter

### PR DESCRIPTION
**Pull request by gish**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export function numberIsFinite(value) {
-  return typeof value === 'number' && isFinite(value);
+  return typeof value === 'number' && Number.isFinite(value);
```
Comment (bstream) [inline src/utils.js]: This was done intentionally to not break IE (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite#Browser_compatibility)

**Comment by bstream** [comment]
This fixes #626

File: src/utils.js (hunk 1 above)
Comment (gish) [reply src/utils.js]: Fixed in fcc9189

**Review by bstream — APPROVED** [review]
lgtm 👌

**Pull request merged**